<!DOCTYPE html>
<html lang="en">
<head>
  		<link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
		<link rel="stylesheet" type="text/css" href="../../theme/css/style.css" />
		
		<script type="text/javascript" src="../../theme/js/jquery-3.4.1.min.js"></script>
		<script type="text/javascript" src="../../theme/js/theme.js"></script>
		
		<title>Bailee Egan - About2</title>
		<meta charset="utf-8" />
		<link href="https://baileeegan.github.io/feeds/all.atom.xml" type="application/atom+xml" rel="alternate" title="Bailee Egan Full Atom Feed" />


</head>

<body id="index" class="home">
		<div id = "menu">

			<div id = "menu-wrapper">
				<header id="banner" class="body menu-flex">
					<a href="../../">Bailee Egan</a>
				</header>
				<div class="menu-flex">
					<a href="https://github.com/BaileeEgan" target="_blank"><img width="20" height="20" src = '../../theme/images/GitHub-Mark-Light-32px.png'/></a>
					<span style="font-size: 24px;">&nbsp;-&nbsp;</span>
					<a href="https://www.linkedin.com/in/bailee-egan" target="_blank"><img width="20" height="20" src = '../../theme/images/In-White-34.png'/></a>
				</div>

						<div class="menu-flex menu-link"><a href="../../pages/about/">About</a></div>
						<div class="menu-flex menu-link"><a href="../../pages/art/">Art</a></div>
						<div class="menu-flex menu-link"><a href="../../pages/projects/">Projects</a></div>
						<div class="menu-flex menu-link"><a href="../../pages/smmsa/">SMMSA</a></div>
				<!---->
			</div>


			<div id = "footer_block" class="body">
					<address id="about" class="vcard body">
					Powered by <a href="http://getpelican.com/">Pelican</a> and <a href="http://python.org">Python</a>.
					</address><!-- /#about -->
			</div>

		</div>
		<div id = "wrapper">
			<div id = "main_content">
    <h1>About2</h1>
    

    <p>My name is Bailee Egan, and I am a bioinformatician by training with additional interests in data science and software development.</p>
<p>I received my M.S. in biology in 2022 at the University of Nebraska-Lincoln. Under Dr. Etsuko Moriyama, I studied corn rootworm chemoreceptors, learning many of the tools and techniques of bioinformatics. In my undergrad, I worked in the lab of Dr. Hope Hollocher at the University of Notre Dame, studying variation in the oral and gut microbiomes of long-tailed macaques.</p>
<p>I have a passion for coding. In my work, I mainly use Bash, Python, and R for processing data. It is also no surprise that data visualization is one of my favorite parts of my work. In my personal projects, I've also learned Javascript, HTML, and CSS for developing small applets and tools, such as a rubric application for grading and a multiple sequence alignment viewer (SMMSA). I have also used the Godot Game Engine for experimenting with small games, shaders, and apps. Using Godot and Rust, I built a 3D terrain editing application.</p>
<h2>Skills</h2>
<ul>
<li>Languages: Python, R, Rust, Javascript (Jquery, React), CSS, HTML</li>
<li>Linux environments, HPC clusters, and job systems (Slurm, PBS)</li>
<li>Python and R packages: pandas, Plotly, Dash, tidyverse/ggplots, Shiny, Rmarkdown</li>
<li>Workflow, package management, package deployment tools: Snakemake, Nextflow,
Conda, Docker, git</li>
</ul>
<h2>Work experiences</h2>
<p><strong>Bioinformatician at Booz Allen Hamilton, June 2023 - present</strong></p>
<ul>
<li>Serves as a bioinformatics subject matter expert for technical support in building biological databases and software</li>
<li>Assists in data curation by preparing test data sets as well as obtaining and organizing data from a variety of different sources, including online databases</li>
</ul>
<p><strong>Bioinformatician at Amentum, October 2022 - June 2023</strong></p>
<ul>
<li>Developed and ran Snakemake workflows for next-generation sequencing data for microbial forensics, transcriptomics, and variant calling analyses</li>
<li>Created and optimized data summarization and visualization tools to streamline and speed up results reporting</li>
<li>Provided bioinformatics support for domestic and deployed laboratories supporting the client</li>
<li>Trained team and customer personnel on use of custom bioinformatics workflows</li>
<li>Collaborated with team to design and establish a genomics and proteomics focused BSL-2
laboratory to support characterization of complex samples</li>
<li>Wrote and revised Standard Operating Procedures (SOPs) to summarize bioinformatics
workflow installation, implementation, and data summarization</li>
</ul>
<p><strong>Bioinformatician at MRIGlobal, August 2022 - October 2022</strong></p>
<ul>
<li>Aided in the development and testing of bioinformatics workflows and VBA scripted Excel macros for data reporting with an emphasis on standardizing the deep characterization of metagenomic samples</li>
<li>Assisted in the deployment of a codon optimization tool using Conda and Docker</li>
</ul>
<h2>Research</h2>
<p><strong>Masterâ€™s thesis, August 2019 - July 2022</strong></p>
<p>Transcriptome assembly and characterization of chemoreceptors for corn rootworms</p>
<ul>
<li>Produced transcriptome assemblies from RNA-seq data using a variety of de novo and genome-guided assembly methods and examined the most optimal methods of assembly based on basic metrics and assembly evaluation tools</li>
<li>Identified novel chemoreceptors in transcriptome assemblies using BLAST search and performed phylogenetic analysis of chemoreceptor genes across insects gathered from literature and online databases</li>
<li>Analyzed chemoreceptor expression by performing differential gene-expression analysis using kallisto, DESeq2, and edgeR</li>
<li>Developed custom visualizations to display statistical results and complex datasets using R libraries</li>
</ul>
<p><strong>Undergraduate biology honors thesis, 2016-2019</strong></p>
<p>Investigating associations between the prokaryotic and eukaryotic symbiotic communities in long-tailed macaques</p>
<ul>
<li>Prepared 16S rRNA sequences from fecal and saliva samples and processed Illumina HiSeq reads using custom USEARCH pipeline to generate family-level read counts</li>
<li>Characterized microbiome community data using alpha and beta diversity methods and
identified taxa strongly associated with certain microbiome features in R</li>
</ul>

			</div>
		</div>

		<div id="modal-box" class="modal">
			<!-- Modal content -->
			<div class="modal-content">
				<span class="modal-close">&times;</span>
				<img src = '' />
			</div>

		</div>

</body>

</html>